presto-client/src/main/java/com/facebook/presto/client/StatementStats.java

```diff
@@ -44,6 +45,7 @@
     private final long processedBytes;
     private final long peakMemoryBytes;
     private final StageStats rootStage;
+    private final DataSize spilledDataSize;
```

Comment:
Let's move this up to be directly after `peakMemoryBytes`, since it is related and should be next to the other memory stats. Put the field, constructor argument, assignment, getter, and `toString`, all in the same order. And update the builder as well.

Also, while in general it's good to use `DataSize`, we should be consistent with the other stats in this class. So, please change it to `long spilledBytes`.

Reply:
@anoopj : Does not look like this is resolved. Please fix this.

Reply:
@raghavsethi 

I believe I fixed it. Are you looking at the [latest diff](https://github.com/prestodb/presto/pull/11910/files#diff-5d764bf71dd84521d4d24b7256b536b2)? Please let me know if I've missed anything.

Reply:
I can still see `spilledDataSize` when I look at the commit.
<img width="998" alt="screen shot 2018-12-20 at 2 43 37 pm" src="https://user-images.githubusercontent.com/715788/50275578-ae0ff480-0465-11e9-80dc-41c8c2c77a2a.png">

Reply:
@raghavsethi Sorry, I missed the part about renaming the variable. I had only changed the type, not the name. I've fixed it. Please let me know if there is any other feedback.